src/splat/util/options.py

```diff
@@ -220,6 +220,10 @@ class SplatOpts:
     detect_redundant_function_end: bool
     # Don't skip disassembling already matched functions and migrated sections
     disassemble_all: bool
+    # Allow specifying that the global segment may be larger than what was automatically detected.
+    # Useful for projects where splat is used in multiple individual files, meaning the expected global segment may not be properly detected because each instance of splat can't see the info from other files.
+    global_vram_start: Optional[int]
```

Comment:
okay I think I get it now with the name of these options. but is global memory always contiguous? I thought sometimes it's not (meaning it's a series of ranges)